### DISCUSSION TITLE
Feature Request: Monthly Income Statement

### DISCUSSION THREAD

**Discussion by AgileSenthil** [ananthakumaran/paisa#295] [Ideas]
I need a view to see all my monthly income breakup / details something similar to "Expenses | Monthly" menu or "Cash Flow | Income Statement" with month wise breakup. This is to validate whether I have received and accounted all the payments due to me. Example: I have interest paying instruments such as Bonds, Deposits and few Rental income. I would like to see whether I received all of them and any TDS deducted. Thanks in advance.

**Comment by obsilva** [comment]
It sounds like the [Income tab](https://demo.paisa.fyi/income) is what you need, have you checked it?
